index.html

```diff
@@ -255,25 +303,44 @@ <h3>
         </h3>
         <p>
           By default, the API always attempts to return a cached position so
-          long as it has a previously acquired position. In this example, we
-          accept a position whose age is no greater than 10 minutes. If the
-          user agent does not have a fresh enough cached position object, it
-          automatically acquires a new position.
+          long as it has a previously acquired position <ins cite="#a5">that
+          matches the requested `accuracyMode`</ins>. In this example, we
+          accept a <ins cite="#a5">precise</ins> position whose age is no
```

Comment:
Reference the [=precise position=] definition

    <del cite="#a5">precise position</del><ins cite="#5">[=precise position=]</ins>